vendor/knative.dev/client/pkg/kn/plugin/stat_windows.go

```diff
@@ -0,0 +1,24 @@
+// Copyright © 2019 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package plugin
+
+import (
+	"os"
+)
+
+// statFileOwner is a no-op on windows
```

Comment:
Don’t you have to put `//+build windows` or similar to this file?

Reply:
I know this is vendored... but perhaps a question for @rhuss ?

Reply:
@maximilien : Nope. The build constraint can also be achieved by file suffixes as you can see `_windows.go`.

Reply:
OK @navidshaikh cool cool. I thought you needed both. Thx.